RecodoDesktop/Recodo.Desktop.Logic/RecorderService.cs

```diff
@@ -84,11 +86,26 @@ public void ResumeRecording()
             }
         }
 
-        public void StopRecording()
+        public async void StopRecording()
```

Comment:
If async, then only Task, not void
void is used only for events (like button click)